src/types/coreBuildService.ts

```diff
@@ -95,12 +95,22 @@ export type ReleasePlanKind = K8sResourceCommon & {
 };
 
 export type ReleasePlanSpec = {
-  displayName: string;
   application: string;
   target: string;
+  data?: any;
+  pipelineRef?: {
+    resolver: ResolverType;
+    params: Param[];
+  };
+  serviceAccount?: string;
   releaseStrategy?: string;
```

Comment:
releaseStrategy no longer exists

Reply:
Thanks, removed. We will also have a separate ticket to remove all references to release strategy from the UI.